backend/src/main/resources/storage.json

```diff
@@ -0,0 +1,502 @@
+[
```

Comment:
Falls diese Datei auch weiterhin so generische Daten enthalten soll bitte ich Dich, diese Daten lieber im Java-Code zu generieren. (For-Schleife)